app/bundles/PluginBundle/EventListener/PushToIntegrationTrait.php

```diff
@@ -54,11 +54,8 @@ protected static function pushIt($config, $lead, &$errors)
         $services = static::$integrationHelper->getIntegrationObjects($integration);
         $success  = true;
 
-        /**
-         * @var string
-         * @var AbstractIntegration $s
-         */
         foreach ($services as $s) {
+            /** @var AbstractIntegration $s */
             $settings = $s->getIntegrationSettings();
```

Comment:
It seems the `$s` has no type before this PR, that's why there is 9 new PHPStan errors (all from one trait, triplicated).
Those are false positive, as those method are actually optional, only defined in the `@method` magic annotation.

That's why the baseline rebuild was needed.